records_mover/records/schema/field/__init__.py

```diff
@@ -152,6 +152,39 @@ def to_sqlalchemy_type(self,
     def to_sqlalchemy_column(self, driver: 'DBDriver') -> Column:
         return field_to_sqlalchemy_column(self, driver)
 
+    def cast_series_type(self, series: 'Series') -> 'Series':
+        import pandas as pd
+        if self.field_type == 'time':
+            if series.size > 0:
+                # https://stackoverflow.com/questions/34501930/how-to-convert-timedelta-to-time-of-day-in-pandas
+                #
+                # Some databases (e.g., MySQL) contains a TIME type
+                # which is ambiguous - it can either represent a
+                # particular time of day or it can represent an
+                # elapsed amount of time.
+                #
+                # Clever, right?
+                #
+                # Unfortunately, Pandas knows about time deltas, but
+                # not about times of day, so upon use of read_sql(),
+                # these objects will come out as as a timedelta64[ns]
+                # type.
+                #
+                # Since that's not what our 'time' field type means,
+                # we have to convert it back to a string, or when it
+                # gets turned into a CSV later, it'll look really
+                # goofy - 1pm will come out as: "0 days 01:00:00".
+                #
+                if type(series[0]) == pd.Timedelta:
+                    # Convert from "0 days 12:34:56.000000000" to "12:34:56"
+                    return series.astype(str).str.split().str[-1].str.split('.').str[0]
+            elif self.field_type == 'date':
+                # TODO fill in here
+                # TODO what about other types?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker